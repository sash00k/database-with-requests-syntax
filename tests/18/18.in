get records where student_id in (get records where students is set).students