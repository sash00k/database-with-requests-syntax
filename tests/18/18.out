Student Иванов С. В. from group 22129, ММФ
Student Петрова И. К. from group 19301, ФФ
Student Зуброва Д. А. from group 8126, ММФ
Student Иванова В. В. from group 22129, ММФ
Student Ударников В. П. from group 17121, ММФ. Also teaches Программирование-2 in 1 groups and mentors 2 students.